src/components/CalculatorPage.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/extensions */
+import React from 'react';
+import styles from './styles/CalculatorPage.module.css';
+import Calculator from './Calculator';
```

Comment:
Great job so far 👍. Kindly check the path, 'Calculator.js' does not match the corresponding name on disk: './src/components/calculator.js'.

If you change the "./Calculator" to  "./calculator" as you have it on your disk, your app will work fine. 📘 😊

Reply:
Thanks you so much for the suggestion @Lizdev-05